backend/src/main/java/de/neuefische/cgnjava222/productgallery/model/ProductListType.java

```diff
@@ -0,0 +1,18 @@
+package de.neuefische.cgnjava222.productgallery.model;
+
+import org.springframework.data.annotation.Id;
+
+import javax.validation.constraints.NotNull;
+
+public record ProductListType(
+        @Id
+        @NotNull
+        String id,
+        @NotNull
+        String title,
+        @NotNull
+        String pictureUrl,
+        @NotNull
+        Integer price
+) {
+}
```

Comment:
Die geschweiften bitte in einer Zeile(18) und die runde in darüber die Zeile(17)

Reply:
das macht er sofort rückgängig, wenn ich autoformat drücke :D